docs/FAQ/FAQ.qmd

```diff
@@ -27,7 +27,32 @@ eatRep::pool.R2(r2)
 
 ## 2. DIF-Richtung in Conquest/eatModel: Was bedeuten negative Werte in der "estDif"-Spalte?
 
-In eatModel wird nur der erste Teil der zweiten Tabelle item\*\[DIFvariable\] aus Conquest übertragen. Das bedeutet, dass negative Werte in der eatModel-estDif-Spalte dafür stehen, dass die jeweiligen Items **in der ersten Gruppe** der \[DIFvariable\] **leichter** sind. Die erste Gruppe ist die mit dem numerisch kleineren Gruppenindikatorwert.
+In eatModel wird nur der erste Teil der zweiten Tabelle $item\times [DIFvariable]$ aus Conquest übertragen. Das bedeutet, dass negative Werte in der eatModel-estDif-Spalte dafür stehen, dass die jeweiligen Items **in der ersten Gruppe** der \[DIFvariable\] **leichter** sind. Die erste Gruppe ist die mit dem numerisch kleineren Gruppenindikatorwert. Zur Illustration ein Beispiel, das Daten aus dem TAM-Paket verwendet:
+
+```{r eval = TRUE}
+library(kableExtra)
+library(eatTools)
+library(eatModel)
+data(data.ex08, package="TAM")
+dat <- data.frame ( id = 1:nrow(data.ex08[["resp"]]), female = unlist(data.ex08[["facets"]])-1, data.ex08[["resp"]])
+def <- defineModel(dat, items= -c(1:2), id="id", DIF.var = "female", software="tam")
+run <- runModel(def)
+res <- getResults(run)
+item<- itemFromRes(res)
+```
+
+Das einzige Item mit B-DIF entsprechend der ETS-Kriterien ist Item Nummer 6: 
+
+```{r eval = TRUE}
+kbl(subset(roundDF(item,digits = 3), item == "I0006")[,c("item", "itemP", "est", "estDif", "absDif", "difIndex", "ETS")], row.names=FALSE) %>% 
+    kable_styling(bootstrap_options = "striped", full_width = FALSE, position = "center")
+```
+
+Die allgemeine Lösungswahrscheinlichkeit des Items ist `r round(100*item[which(item[,"item"] == "I0006"),"itemP"], digits = 1)` Prozent. Der positive DIF-Wert bedeutet, dass das Item in der ersten Gruppe (also female = 0, die Gruppe der Jungen) schwerer ist. Die Lösungshäufigkeit sollte also für female = 0 geringer sein, als für female = 1: 
+
+```{r eval = TRUE}
+tapply(dat[,"I0006"], dat[,"female"], mean)
```

Comment:
NIT: I would have expected a demonstration with repect to the group specific logit value. Could you add information about this? I.e. how the DIF value adds to the logit for both groups and how this is related to the DIF effect?